docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,273 +1,378 @@
+   ```
+
+### Enroll a server with the `env:local-prod` label
+
+1. Open a separate terminal.
+
+1. Create a join token for the server to use to join the cluster:
+
+   ```code
+   $ tctl tokens add --type=node
+   ```
+
+1. Assign the token to <Var name="token" />.
+
+1. Copy the following YAML document to a file called `local-prod.yaml`. This
+   configuration starts the Teleport SSH Service with the label
+   `env:local-prod`:
+
+   ```yaml
+   version: v3
+   teleport:
+     data_dir: /var/lib/teleport
+     join_params:
+       token_name: <Var name="token" />
+       method: token
+     proxy_server: <Var name="example.teleport.sh:443" />
+   auth_service:
+     enabled: "false"
+   ssh_service:
+     enabled: "true"
+     labels:
+       env: local-prod
+   proxy_service:
+     enabled: "false"
+   ```
+
+1. Assign the absolute path to the configuration file you created to 
+   <Var name="prod-config-path" />.
+
+1. Start the Teleport SSH Service on a local Docker container with the
+   configuration and join it to your cluster:
+
+   ```code
+   $ docker run -v <Var name="prod-config-path" />:/etc/teleport/teleport.yaml (=teleport.latest_ent_debug_docker_image=)
+   ```
+
+1. Ensure that the server has joined the cluster:
+
+   ```code
+   $ tsh ls env=local-prod
+   Node Name    Address    Labels
+   ------------ ---------- --------------
+   ba2290caf694 ⟵ Tunnel   env=local-prod
+   ```
+
+## Step 2/3. Create a Teleport role
+
+Create a role that can access servers with the `env:local-dev` label but not the
+`env:local-prod` label.
+
+1. Create a file called `role.yaml` with the following content:
+
+   ```yaml
+   kind: role
+   version: v7
+   metadata:
+     name: local-dev-only
+   spec:
+     allow:
+       logins: [root]
+       node_labels:
+         env: local-dev
+   ```
 
-## Step 1/3. Add local users with preset roles
+   The `allow` block indicates what the user is allowed to access. By default,
+   nothing is allowed, so each role must include at least one `allow` field to
+   provide permissions.  
 
-Teleport provides several preset roles:
+   The `spec.allow.logins` field allows the user to assume the `root` login when
+   connecting to a server. You can change this to a less permissive login, but
+   we are using `root` because it is the only available login on the Docker
+   containers we spun up.
 
-(!docs/pages/includes/preset-roles-table.mdx!)
+   `spec.allow.node_labels` uses wildcard syntax, which matches one or more
+   characters, to allow users to connect to any server with a label that begins
+   `env:local-`, such as the `env:local-dev` and `env:local-prod` labels we
+   assigned to our servers.
 
-<Tabs>
-<TabItem scope={["oss"]} label="Teleport Community Edition">
-Invite the local user Alice as cluster `editor`:
+   However, since the `deny.node_labels` field specifies `env:local-prod`, a
+   user with this role would only be able to access the server with the
+   `env:local-dev` label.
```

Comment:
i'm not suggesting adding it, but the example role doesn't actually contain any `deny` section, we could just say they would only be allowed to access nodes with `env:local-dev`